contrib/imgts/lib_entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
As a courtesy to future readers, could you chmod this 644? That gives an immediate indication that the file is intended to be sourced, not executed. (So does the `lib` in the name, but this way there are no inconsistencies).

Reply:
Oh good catch, yes I normally prefer that as well.